packages/ui-components/src/utilities/Focus.ts

```diff
@@ -0,0 +1,5 @@
+import { getFirstFocusable, getLastFocusable, setFocusVisibility } from '@fluentui/react';
```

Comment:
Why do we need the re-export under a different name? It is especially confusing to me that the functions are renamed so that they sound like objects.

Reply:
It was to avoid import from @fluentui in extensions. Would renaming to `getUIFirstFocusable` or exporting same with same name `getFirstFocusable ` fine?

Reply:
Gotcha, so you are kind of facading it. Yes, I'd prefer `getUIFirstFocusable`